### PR TITLE
Teams Page

### PR DESCRIPTION
**Pull request by ayushshah381**

### CODE REVIEW

**Review by shahp98 — CHANGES_REQUESTED** [review]

File: Ayush Shah/src/src/components/HackathonForm.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import '../css/buddy.css';
```
[the remark this answers is not in the document]
Comment (Kushald2805) [reply Ayush Shah/src/src/components/HackathonForm.js]: Buddy doesn't have CDN

[the remark this answers is not in the document]
Comment (shahp98) [reply Ayush Shah/src/src/components/HackathonForm.js]: https://twitter.com/BuddyCSS/status/1029349288655110144

[the remark this answers is not in the document]
Comment (shahp98) [reply Ayush Shah/src/src/components/HackathonForm.js]: Apparently it does. In the latest version, they've added CDN Links.

[the remark this answers is not in the document]
Comment (shahp98) [reply Ayush Shah/src/src/components/HackathonForm.js]: `<link rel="stylesheet" type="text/css" href="https://buddycss-cc55.kxcdn.com/downloads/1.0.7/css/buddy.plugins.min.css">
<link rel="stylesheet" type="text/css" href="https://buddycss-cc55.kxcdn.com/downloads/1.0.7/css/buddy.min.css">
<script type="text/javascript" src="https://buddycss-cc55.kxcdn.com/downloads/1.0.7/js/buddy.min.js">`

[the remark this answers is not in the document]
Comment (shahp98) [reply Ayush Shah/src/src/components/HackathonForm.js]: `<link rel="stylesheet" type="text/css" href="https://buddycss-cc55.kxcdn.com/downloads/1.0.7/css/buddy.plugins.min.css">
<link rel="stylesheet" type="text/css" href="https://buddycss-cc55.kxcdn.com/downloads/1.0.7/css/buddy.min.css">
<script type="text/javascript" src="https://buddycss-cc55.kxcdn.com/downloads/1.0.7/js/buddy.min.js">`

[the remark this answers is not in the document]
Comment (paceite) [reply Ayush Shah/src/src/components/HackathonForm.js]: The problem is we have to make a few changes in the buddy files itself...that won't be possible using cdn

**Comment by Kushald2805** [comment]
Push the latest code please

**Comment by paceite** [comment]
Yes sure.

**Comment by Kushald2805** [comment]
You have missed out Accept and Decline options on the card of Teams Page. Rest everything is perfect.

**Comment by paceite** [comment]
> You have missed out Accept and Decline options on the card of Teams Page.
> Search bar is missing too.

> You have missed out Accept and Decline options on the card of Teams Page.
> Search bar is missing too.

Oh shit yes...Accept and Decline part I saw that now...
Search bar I tried the design but it wasn't coming properly so I had left that out..will try it now

**Comment by Kushald2805** [comment]
Count on the top right corner is missing

**Comment by paceite** [comment]
I'll check that and rectify it tomorrow

**Comment by paceite** [comment]
I have added the count part....will resume figuring out multiple select form

**Pull request closed without merging**